cfme/utils/appliance/__init__.py

```diff
@@ -217,7 +218,7 @@ def configure_appliance_internal(self, region, dbhostname, username, password, d
         self._run("--region {region} --internal --hostname {dbhostname} --username {username}"
             " --password {password} --dbname {dbname} --verbose --dbdisk {dbdisk}".format(
                 region=region, dbhostname=dbhostname, username=username, password=password,
-                dbname=dbname, dbdisk=dbdisk))
+                dbname=dbname, dbdisk=dbdisk), timeout=4 * 60)
```

Comment:
Why pass 4*60 and not 240 here?

Reply:
4*60 is more readable. but I'd rather advice using string value '4m'  if there is wait_for behind.

Reply:
@izapolsk it is not possible to pass "4m". Float is required.